test/lib/ansible_test/_data/requirements/ansible.txt

```diff
@@ -12,4 +12,4 @@ packaging
 # NOTE: Ref: https://github.com/sarugaku/resolvelib/issues/69
 # NOTE: When updating the upper bound, also update the latest version used
 # NOTE: in the ansible-galaxy-collection test suite.
-resolvelib >= 0.5.3, < 0.9.0  # dependency resolver used by ansible-galaxy
+resolvelib >= 0.5.3, < 0.10.0  # dependency resolver used by ansible-galaxy
```

Comment:
Since `resolvelib` is pre-installed in our test containers, we should temporarily pin the latest version allowed to force the tests to run with that version. Once the tests have passed that commit can be reverted.

Please make those changes without force pushing, so that we keep the reference to the passing CI run. We can squash the commits when merging the PR so the temporary commits won't be in the final commit merged to the `devel` branch.


```suggestion
resolvelib == 0.9.0  # dependency resolver used by ansible-galaxy
```

Reply:
https://github.com/ansible/ansible/pull/79399/commits/0d3544bacab2935a90827dfd25fa7cf7e9613899 already pass CI.

@mattclay your suggested change also commit as additional for trigger CI, too.